src/cmds/orgs:create.ts

```diff
@@ -26,6 +26,26 @@ export const builder = {
   },
 };
 
+export async function createOrg(
```

Comment:
Since we are calling this in a row with other commands we want the try / catch in this function too so we get the same nice errors.